core/src/toga/__init__.py

```diff
@@ -1,139 +1,94 @@
 from __future__ import annotations
 
+import importlib
+import importlib.util
 import warnings
-from pathlib import Path
 
-from .app import App, DocumentApp
-from .colors import hsl, hsla, rgb, rgba
-from .command import Command, Group
-from .dialogs import (
-    ConfirmDialog,
-    ErrorDialog,
-    InfoDialog,
-    OpenFileDialog,
-    QuestionDialog,
-    SaveFileDialog,
-    SelectFolderDialog,
-    StackTraceDialog,
-)
-from .documents import Document
-from .fonts import Font
-from .icons import Icon
-from .images import Image
-from .keys import Key
-from .types import LatLng, Position, Size
-from .widgets.activityindicator import ActivityIndicator
-from .widgets.base import Widget
-from .widgets.box import Box
-from .widgets.button import Button
-from .widgets.canvas import Canvas
-from .widgets.dateinput import DateInput, DatePicker
-from .widgets.detailedlist import DetailedList
-from .widgets.divider import Divider
-from .widgets.imageview import ImageView
-from .widgets.label import Label
-from .widgets.mapview import MapPin, MapView
-from .widgets.multilinetextinput import MultilineTextInput
-from .widgets.numberinput import NumberInput
-from .widgets.optioncontainer import OptionContainer, OptionItem
-from .widgets.passwordinput import PasswordInput
-from .widgets.progressbar import ProgressBar
-from .widgets.scrollcontainer import ScrollContainer
-from .widgets.selection import Selection
-from .widgets.slider import Slider
-from .widgets.splitcontainer import SplitContainer
-from .widgets.switch import Switch
-from .widgets.table import Table
-from .widgets.textinput import TextInput
-from .widgets.timeinput import TimeInput, TimePicker
-from .widgets.tree import Tree
-from .widgets.webview import WebView
-from .window import DocumentMainWindow, MainWindow, Window
+toga_core_imports = {
```

Comment:
This list is a bit difficult to parse; it's not alphabetically sorted; it's not preserving the historical sort order; and it's not providing any sort to conceptual ordering. 

I think it would benefit from preserving the historical sort order (i.e., sorted by the full import path of the original module), with interleaved comments highlighting that ordering.